dataprofiler/tests/profilers/profiler_options/test_num_quantiles_option.py

```diff
@@ -0,0 +1,142 @@
+
+        # Valid configurations
+        option = self.get_options(num_quantiles=50)
+        self.assertEqual([], option._validate_helper())
+        option = self.get_options(num_quantiles=2000)
+        self.assertEqual([], option._validate_helper())
+        option = self.get_options(num_quantiles=1)
+        self.assertEqual([], option._validate_helper())
+
+        # Option num_quantiles
+        option = self.get_options(num_quantiles="Hello World")
+        expected_error = [f"{optpth}.num_quantiles must be a positive integer."]
+        self.assertSetEqual(set(expected_error), set(option._validate_helper()))
+
+        # Option num_quantiles cannot be a float, must be an int
+        option = self.get_options(num_quantiles=1.1)
+        expected_error = [f"{optpth}.num_quantiles must be a positive integer."]
+        self.assertSetEqual(set(expected_error), set(option._validate_helper()))
+
+        # Option num_quantiles must be a positive integer
+        option = self.get_options(num_quantiles=0)
+        expected_error = [f"{optpth}.num_quantiles must be a positive integer."]
+        self.assertSetEqual(set(expected_error), set(option._validate_helper()))
+
+        # Option num_quantiles must be a positive integer
+        option = self.get_options(num_quantiles=-5)
+        expected_error = [f"{optpth}.num_quantiles must be a positive integer."]
+        self.assertSetEqual(set(expected_error), set(option._validate_helper()))
+
+    def test_validate(self):
+        super().test_validate()
+
+        optpth = self.get_options_path()
+
+        # Default configuration
+        option = self.get_options(num_quantiles=1000)
+        self.assertEqual([], option._validate_helper())
+
+        # Valid configurations
+        option = self.get_options(num_quantiles=50)
+        self.assertEqual([], option._validate_helper())
+        option = self.get_options(num_quantiles=2000)
+        self.assertEqual([], option._validate_helper())
+        option = self.get_options(num_quantiles=1)
+        self.assertEqual([], option._validate_helper())
+
+        # Option num_quantiles cannot be a string, must be an int
+        option = self.get_options(num_quantiles="Hello World")
+        expected_error = (
+            "NumQuantilesOption.num_quantiles must be a " "positive integer"
+        )
+        with self.assertRaisesRegex(ValueError, expected_error):
+            option.validate()
+
+        # Option num_quantiles cannot be a float, must be an int
+        option = self.get_options(num_quantiles=1.1)
+        expected_error = (
+            "NumQuantilesOption.num_quantiles must be a " "positive integer"
+        )
+        with self.assertRaisesRegex(ValueError, expected_error):
+            option.validate()
+
+        # Option num_quantiles must be a positive integer
+        option = self.get_options(num_quantiles=0)
+        expected_error = (
+            "NumQuantilesOption.num_quantiles must be a " "positive integer"
```

Comment:
if you accept, https://github.com/capitalone/DataProfiler/pull/896/files#diff-00da4b3383cf04d1ea70e7d179d78433f0bfde3ac6909c791eb92363c5a34f3bR302 

you'll need to update this line